pages/css-in-js/api.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import withRoot from 'docs/src/modules/components/withRoot';
 import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
+import requireMarkdown from 'docs/src/modules/utils/requireMarkdown';
```

Comment:
It's a lot of pages to update. Some of the alternatives:
- We add an error boundary on the _app.js or withRoot level to catch the error. Maybe remove the cookie and reoad.
- Prevent the deployment of missing translations in the first place.

Reply:
That was my initial idea too but [error boundaries do not work for SSR](https://reactjs.org/docs/error-boundaries.html#introducing-error-boundaries).

While this is mainly aimed at missing translation I think it's good to have a fallback page in general so that people are not presented with a "white screen of death". I'm also not sure how this would work. Just remove `props.lang`? How would that work if there are translations for Chinese but not Portuguese?

Reply:
We don't SSR the docs.

Reply:
Removing the lang cookie and hard reloading can work.

Reply:
We don't have Portuguese translations yet.

Reply:
@oliviertassinari But they're already planned so why ignore them?